plugins/flashrom/README.md

```diff
@@ -72,6 +72,7 @@ This plugin uses the following plugin-specific quirks:
 | Quirk                  | Description                                 | Minimum fwupd version |
 |------------------------|---------------------------------------------|-----------------------|
 |`FlashromProgrammer`    | Used to specify the libflashrom programmer to be used.     | 1.5.9                 |
+|`FlashromLspconAuxDeviceName` | sysfs name of the drm_dp_aux_dev device over which to read LSPCON version | 1.6.1 |
```

Comment:
This doesn't feel very flashrom, no?

Reply:
Changed to `ParadeLspcon`. I had chosen to use the plugin name prefix, but clearly specifying the device type seems fine too.